scripts/aws/cfn/once4account/dev.yaml

```diff
@@ -56,6 +61,113 @@ Resources:
               ArnEquals: 
                 aws:SourceArn: !Ref AlarmSNSTopic   
 
+  # - Alarm message lambda subscription
+  AlarmSNSTopicToLambda:
+    Condition: HasCoreAccount
+    Type: AWS::SNS::Subscription
+    Properties:
+      Protocol: lambda
+      Endpoint: !GetAtt LambdaFunction.Arn
+      Region: !Ref AWS::Region
+      TopicArn: !Ref AlarmSNSTopic
+
+  LambdaFunctionPermission:
+    Type: AWS::Lambda::Permission
+    Condition: HasCoreAccount
+    Properties:
+      Action: lambda:InvokeFunction
+      FunctionName: !GetAtt LambdaFunction.Arn
+      Principal: sns.amazonaws.com
+
+  LambdaFunctionExecutionRole:
+    Type: AWS::IAM::Role
+    Condition: HasCoreAccount
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: [lambda.amazonaws.com]
+            Action: ['sts:AssumeRole']
+      Path: /
+      Policies:
+        - PolicyName: PublisherPolicy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - logs:CreateLogGroup
+                Resource:
+                  - !Sub 'arn:aws:logs:*:${AWS::AccountId}:log-group:/aws/lambda/*'
+              - Effect: Allow
+                Action:
+                  - logs:CreateLogStream
+                  - logs:PutLogEvents
+                Resource:
+                  - !Sub 'arn:aws:logs:*:${AWS::AccountId}:log-group:*'
+              - Effect: Allow
+                Action:
+                  - sns:Publish
+                Resource:
+                  - !Sub arn:${AWS::Partition}:sns:${AWS::Region}:${PnCoreAwsAccountId}:pn-AllAlarmSnsTopic
+
+  LambdaFunction:
```

Comment:
Cosa succede se questa lambda, per caso particlarissimo, va in errore ? 
Secondo me non c'è nulla che ci avvisi.